### PR TITLE
Add system exit code for incorrect command line arguments(#1324)

### PR DESCRIPTION
**Pull request by joshua-jin**
fix for issue #1324

### CODE REVIEW

**Comment by nehashri** [comment]
@gaugebot recheck the pull request.

**Comment by nehashri** [comment]
@joshua-jin  Thanks for your pull request! Looks like this is your first contribution to Gauge. Before we can look at your pull request, you'll need to sign our Contributor License Agreement (CLA).
Please visit https://gauge-bot.herokuapp.com/cla/ to read and sign it.

**Comment by joshua-jin** [comment]
@gaugebot recheck the pull request.

File: gauge.go [hunk 1]
```diff
@@ -28,6 +29,7 @@ func main() {
 	defer recoverPanic()
 	if err := cmd.Parse(); err != nil {
 		logger.Info(true, err.Error())
+		syscall.Exit(1)
```
Comment (nehashri) [inline gauge.go]: @joshua-jin We use os.Exit(1) in most places, to exit with a non zero exit code. Doing the same here would help.

Comment (joshua-jin) [reply gauge.go]: Ok, it's done.

**Comment by nehashri** [comment]
@joshua-jin Thanks for you contributions.

**Pull request merged**